src/confirmation-window/components/TxParamValue.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Param, ParamMetaType } from "../../pointsdk/index.d";
+
+type Props = { param: Param };
+
+const TxParamValue = ({ param }: Props) => {
+    switch (param.meta?.type) {
+        case ParamMetaType.ZERO_CONTENT:
+            return <span>no content</span>;
+        case ParamMetaType.STORAGE_ID:
+            return (
+                <a
+                    href={`https://point/_storage/${param.value}`}
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    content in storage &rarr; {param.value}
+                </a>
+            );
+        case ParamMetaType.TX_HASH:
+            return (
+                <span>Blockchain Transaction Hash &rarr; {param.value}</span>
+            );
+        case ParamMetaType.NOT_FOUND:
+            return (
+                <span>
+                    {param.value} <em>(not found in storage nor blockchain)</em>
```

Comment:
This message looks a bit like an error, as if something was not found and hence may be wrong. I would rather say something like `unfamiliar data id`. Probably in the future we will emphasise it even more, but for now we need to figure out how common this case actually is.

Reply:
The less verbose way would be to just say `(?)`

It can be arbitrary binary data that the contract wants to store, doesn't have to exist in our storage necessarily